src/database/functions/GuildSettingsFunctions.ts

```diff
@@ -23,18 +23,19 @@ function GuildScheme(guildID: string) {
         moderationSettings: {
             enable: false,
             modLogChannel: "",
-            modBlackList:[],
-            urlBlock:false,
-            urlWhiteList:[]
+            modBlackList: [],
+            urlBlock: false,
+            urlWhiteList: [],
         },
         welcomeSettings: {
             enable: false,
             welcomeDM: false,
             welcomeChannel: false,
             welcomeChannelID: "",
-            channelMessage: "Welcome {member} to {server}!",
-            dmMessage:"Welcome {member} to {server}!",
+            welcomeChannelMessage: "Welcome {member} to {server}!",
```

Comment:
again you can't just change it in one place you need to change it in like 2 other files

Reply:
ay that, i forgot to change it to channelMessage